src/client/Directory.Build.props

```diff
@@ -35,6 +35,7 @@
     <Authors>Microsoft</Authors>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=844761</PackageProjectUrl>
+    <PackageReleaseNotes>The changelog is available at https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/blob/dev/changelog.txt and the roadmap at https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki#roadmap </PackageReleaseNotes>
```

Comment:
this is pointing to adal's release notes...am i missing something? @jmprieur @bgavrilMS

Reply:
Good catch, Jenny. My bad

Reply:
@jmprieur i fixed it.

Reply:
@jennyf19 thank you.